pyanaconda/payload/__init__.py

```diff
@@ -1307,7 +1313,7 @@ def _setupInstallDevice(self, storage, checkmount):
                 elif method.method == "cdrom":
                     raise PayloadSetupError("no usable optical media found")
 
-        return url, mirrorlist, sslverify
+        return url, mirrorlist, metalink, sslverify
```

Comment:
I would like to see this as some data object or other structure. Like now it is easy to swap arguments with each other.

Reply:
I see your point.
I will remove passing sslverify as it is not needed. The only function calling this one can get the value on its own.
Given there is only single caller of this function, and having (url, mirrorlist, metalink) structure or namedtuple in this module doesn't seem quite right to me, I'll leave consolidating this for future refactoring.

Reply:
Ok, it's fine for me :)